pyproject.toml

```diff
@@ -48,7 +48,7 @@ catboost = ">=0.26.1"
 optuna = "*"
 torch = [
     {platform = "win32", python = "3.6.1", version = "1.7.0"},
-    {version = ">=1.0.0"}
```

Comment:
Check min version with timm and transformers. (1.10?)

Reply:
![image](https://github.com/sb-ai-lab/LightAutoML/assets/70897456/32a43a2e-63d5-43b3-879b-c011d3d691e7)

timm requirements

Reply:
Still.